docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: init
+  labels:
+    app: rucio
+spec:
+  restartPolicy: Never
+  containers:
+    - name: init
+      image: rucio/rucio-init:latest
+      imagePullPolicy: Always
+      env:
+        - name: RUCIO_CFG_DATABASE_DEFAULT
+          value: postgresql://rucio:<PASSWORD>@postgres-postgresql/rucio
+        - name: RUCIO_CFG_DATABASE_SCHEMA
+          value: test
+        - name: RUCIO_CFG_BOOTSTRAP_USERPASS_IDENTITY
+          value: admin
+        - name: RUCIO_CFG_BOOTSTRAP_USERPASS_PWD
+          value: xxx
+        - name: RUCIO_PRINT_CFG
+          value: "true"
+```
+
+And then apply the pod config:
+
+```
+kubectl apply -f init-pod.yaml
+```
+:::warning
+Please notice that in this case, the various credentials will have to be properly stored as secrets. See the Managing Secrets section for more information.
+:::
+
+## Creating a LanDB set
+
+Create a [new LanDB set](https://landb.cern.ch/portal/sets/create), following the recommendations: 
+1. Type: Interdomain
+2. Network Domain: GPN
+3. Responsible: `<your-egroup>`
+4. Description: use the following fields: 
+```
+OPENSTACK_PROJECT=cc059d57-6e98-4688-a3be-aae2b451868b,<your-openstack-project-ID>
+```
+
+:::tip[LoadBalancer tip]
+The ID `cc059d57-6e98-4688-a3be-aae2b451868b` will allow the LoadBalancer as a Service (LBaaS) instance to [assign LBs to this set](https://clouddocs.web.cern.ch/networking/load_balancing.html#adding-load-balancer-to-landb-sets). In the specific case of the COMPASS rucio instance, the `openstack-landb-set-access` is being set as a member of the `rucio-it-admins` egroup.
+Please refer to the LoadBalancers section for more information.
+:::
+![[/img/landb-set-create.png]]
+
+# Populating the cluster
+There are four main components that need to be installed in oder to have the Rucio cluster operative:
+1. Rucio Servers
+2. Rucio Authentication
+3. Rucio Daemons
+4. Rucio Web UI
+
+The following sections are based on the deployment of the [COMPASS Rucio instance](https://gitlab.cern.ch/rucio-it/flux-compass), within the CERN infrastructure.
+## Managing secrets
+### Sealed-Secrets
+[Reference Documentation](https://github.com/bitnami-labs/sealed-secrets?tab=readme-ov-file#installation).
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/sealed-secrets.yaml?ref_type=heads).
+
+A very efficient way of managing secrets in the cluster is Bitnami's Sealed-Secrets. Install the Helm chart by executing:
+```sh
+helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
+rucio-helm-repo.yaml sealed-secrets.yaml
+
+kubectl create namespace sealed-secrets
+
+kubectl apply -f sealed-secrets.yaml 
+```
+
+An example of the `sealed-secrets.yaml` configuration file is provided below:
+```yaml sealed-secrets.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sealed-secrets
+  labels:
+    name: sealed-secrets
+---
```

Comment:
Can these just get split up into different code blocks? Easier to copy and paste that way.